gobgp/cmd/common.go

```diff
@@ -108,10 +108,12 @@ var actionOpts struct {
 	NexthopAction       string `long:"next-hop" description:"specifying a next-hop action of policy"`
 }
 
-var mrtOpts struct {
+type mrtOpts struct {
```

Comment:
I prefer to keep 'var' for consistency. You can use 'mrtOpts' variant globally. The way to use neighborsOpts in gobgp/cmd/neighbor.go would be helpful.

Reply:
Sure. Want me to add rest of the options there (filename, count, skip) ? Having half of options passed and half of it taken from some package variable would seem kinda hard to follow IMO.

Reply:
Surely, let's add them there and handle in the same way. Thnaks!